src/app/app.routes.ts

```diff
@@ -18,9 +18,14 @@
  */
 
 import { Routes, RouterModule } from '@angular/router';
-import { ArlasWuiComponent } from './app.component';
+import { LoginComponent, RegisterComponent, ToolkitComponent } from 'arlas-wui-toolkit';
+import { ArlasWuiRootComponent } from './components/arlas-wui-root/arlas-wui-root.component';
 
 export const ROUTES: Routes = [
```

Comment:
Is it normal that the routes are declared twice, but do not contain the same paths ?

Reply:
fixed